test/config/config.js

```diff
@@ -28,6 +28,11 @@ module.exports = {
     password: process.env.SEQ_MSSQL_PW   || process.env.SEQ_PW   || null,
     host:     process.env.SEQ_MSSQL_HOST || process.env.SEQ_HOST || '127.0.0.1',
     port:     process.env.SEQ_MSSQL_PORT || process.env.SEQ_PORT || 11433,
+    dialectOptions: {
+      instanceName:  process.env.MSSQL_INSTANCE ? process.env.MSSQL_INSTANCE : 'SQLEXPRESS',
+      // big insert queries need a while
```

Comment:
Why empty string? you could just use `instanceName: process.env.MSSQL_INSTANCE`.
Btw this is only for the tests, this is not a breaking change in any way